.spdx-ignore

```diff
@@ -0,0 +1,41 @@
+LICENSE
+*.html
+*.json
+*.md
+*.png
+*.svg
+*.rst
+*.txt
+*.ptx
+*.pyc
+.gitattributes
+.gitignore
+.git-blame-ignore-revs
+requirements*.txt
+
+# Configuration files that shouldn't have SPDX headers
+conda/recipes/*/meta.yaml
+configs/*.yml
+configs/*.yaml
+.pre-commit-config.yaml
+pyproject.toml
```

Comment:
All of the configuration files can and do have SPDX identifiers so we shouldn't exclude them.